app/views/projects/random.html.haml

```diff
@@ -0,0 +1,68 @@
+- content_for :title do
+  = @project.title
+
+.row
+  .col-md-12
+    .page-header.media
+      .pull-left
+        %h3
+          = @project.title
+          %br
+          %small
+            = render :partial => "projects/state_name", :locals => {:project =>  @project }
+            by
+            = link_to @project.originator.name, @project.originator
+      .pull-right
+        %button.btn.btn-default#play-pause-button{ data: { running: :true } }
+          =fa_icon :pause, id: 'play-pause-icon'
+.row
+  .col-sm-8
+    :markdown
+      #{@project.description}
+  .col-sm-4
+    .row
+      .col-sm-12
+        %h5
+          - if @project.users.empty?
```

Comment:
I might be misreading this, but isn't this missing an else? Wouldn't this display 'No Hackers:' if there are no project participants?